package.json

```diff
@@ -96,5 +96,8 @@
   "bugs": {
     "url": "https://github.com/CenterForOpenScience/ember-osf-preprints/issues"
   },
-  "homepage": "https://github.com/CenterForOpenScience/ember-osf-preprints#readme"
+  "homepage": "https://github.com/CenterForOpenScience/ember-osf-preprints#readme",
+  "dependencies": {
+    "unicode-byte-truncate": "^1.0.0"
```

Comment:
should go in devDependencies (see https://stackoverflow.com/questions/42159153/why-does-ember-installs-everything-as-devdependencies-instead-of-normal-dependen/42162627 for explanation of why this makes sense)